microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/ls/java/JavaFileTextDocumentService.java

```diff
@@ -290,8 +289,9 @@ public CompletableFuture<Either<List<? extends Location>, List<? extends Locatio
 									return location;
 								}).collect(Collectors.toList());
 						if (isDefinitionLinkSupport()) {
-							// I don't understand
-							// return Either.forRight(locations);
+							// Uncommented in
+							// https://github.com/redhat-developer/quarkus-ls/issues/499#issuecomment-1021555227

```

Comment:
I think you meant to reference https://github.com/eclipse/lsp4mp/issues/215#issuecomment-984714139 , right ?

Reply:
Ah whoops, yes I do